README.md

```diff
@@ -250,5 +251,151 @@ mstorage.meta("<media_id>", new CompletionHandler<MediaMeta>() {
 });
 ```
 
+### Get specific media metadata
+
+* Exif
+
+```java
+mstorage.meta("<media_id>", "exif", new CompletionHandler<MediaMeta>() {
+    @Override
+    public void onCompleted(Map<String, String> exif) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* Google Photo Sphere XMP
+
+```java
+mstorage.meta("<media_id>", "gpano", new CompletionHandler<MediaMeta>() {
+    @Override
+    public void onCompleted(Map<String, String> exif) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* All user metadata
+
+```java
+mstorage.meta("<media_id>", "user", new CompletionHandler<MediaMeta>() {
+    @Override
+    public void onCompleted(Map<String, String> userMeta) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* Specific user metadata
+
+```java
+mstorage.meta("<media_id>", "user.<key>", new CompletionHandler<MediaMeta>() {
+    @Override
+    public void onCompleted(Map<String, String> userMeta) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+### Attach media metadata
+You can define your original metadata as a 'user metadata'.
+
```

Comment:
You can add the same line as [this](https://github.com/ricohapi/media-storage-js#add-user-metadata-to-a-file).